src/dfx/assets/project_templates/vanilla_js/src/__frontend_name__/assets/.ic-assets.json5

```diff
@@ -1,19 +1,17 @@
 [
-    {
-        "match": "**/*",
+  {
+    "match": "**/*",
+    // Provides a base set of security headers that will work for most dapps.
+    // Any headers you manually specify will override the headers provided by the policy.
+    // See 'dfx info security-policy' to see the policy and for advice on how to harden the headers.
+    // Once you improved the headers for your dapp, set the security policy to "hardened" to disable the warning.
+    // Options are: "hardened" | "standard" | "disabled".
+    "security_policy": "standard",
+    // Uncomment to disable the warning about using the
+    // standard security policy, if you understand the risk
+    // "disable_security_policy_warning": true,
 
-        // Provides a base set of security headers that will work for most dapps.
-        // Any headers you manually specify will override the headers provided by the policy.
-        // See 'dfx info security-policy' to see the policy and for advice on how to harden the headers.
-        // Once you improved the headers for your dapp, set the security policy to "hardened" to disable the warning.
-        // Options are: "hardened" | "standard" | "disabled".
-        "security_policy": "standard",
-
```

Comment:
Right now the remaining changes to these .ic-asset.json files are whitespace-only: changing the indentation, and removing some newlines (newlines that improved readability). Can we make it so this PR doesn't change these files at all?

Reply:
the "standard" policy does not work on safari. I've added an identical CSP to each project currently works on Safari, but that should be updated at a later date when dfx can do the handling of CSP between dev / prod modes